src/download/linux.html

```diff
@@ -33,10 +33,15 @@ <h3>Thank you for downloading the O3DE Debian Package</h3>
                 <p>
                     <a href="https://o3debinaries.org/main/Latest/Linux/o3de_2305_0.deb" class="btn">Download v23.05.0 Release</a><span class="additional-download-url">Additional Downloads: <a href="https://o3debinaries.org/main/Latest/Linux/o3de_2305_0.deb.sha256">SHA256</a> <a href="https://o3debinaries.org/main/Latest/Linux/o3de-releases.gpg">GPG</a></span>
                 </p>
+                
+                <a href="https://snapcraft.io/o3de">
+                    <img alt="Get it from the Snap Store" src="https://snapcraft.io/static/images/badges/en/snap-store-black.svg" />
+                </a>
 				<br/>
 				<p class="small-note">
-					Try out features and fixes currently in development by downloading the <a href="https://o3debinaries.org/development/Latest/Linux/O3DE_latest.deb">Nightly Development Build here</a> (<a href="https://o3debinaries.org/development/Latest/Linux/O3DE_latest.deb.sha256">SHA256</a> <a href="https://o3debinaries.org/main/Latest/Linux/o3de-releases.gpg">GPG</a>). 
-					Note: These nightly builds are created automatically from the latest source code revisions in the O3DE development branch. These builds contain untested features and possible bugs.
+					Try out features and fixes currently in development by downloading the Nightly Development Build here: <a href="https://o3debinaries.org/development/Latest/Linux/O3DE_latest.deb">Debian Package</a> (<a href="https://o3debinaries.org/development/Latest/Linux/O3DE_latest.deb.sha256">SHA256</a> <a href="https://o3debinaries.org/main/Latest/Linux/o3de-releases.gpg">GPG</a>)
+                    or <a href="https://o3debinaries.org/2.1.0/Linux/o3de_2.1.0_amd64.snap">Snap Package</a> (<a href="https://o3debinaries.org/2.1.0/Linux/o3de_2.1.0_amd64.snap.sha384">SHA384</a> <a href="https://o3debinaries.org/main/Latest/Linux/o3de-releases.gpg">GPG</a>). 
```

Comment:
Is there a similar latest tag we can use for the development build for the snap package like we do with the debian package?

Reply:
Requires some changes to the branch tagging for the snaps. I'm working on that in a branch, but the current behavior for development doesn't seem to work with the snap builds, as the branch tag fails to apply to the s3 bucket location. This can wait till I get that fixed, though

Reply:
Works for me, sounds like there's a fix incoming to address. That way we won't have to continually update the tagged version for dev builds. This should work for now.